.goreleaser.yaml

```diff
@@ -13,17 +14,28 @@ builds:
       - arm64
     ldflags:
       - -s -w -X github.com/budimanjojo/talhelper/v3/cmd.version={{.Version}}
-  - id: talhelper-windows-amd64
+  - id: windows
     env:
       - CGO_ENABLED=0
     goos:
       - windows
     goarch:
       - amd64
+      - arm64
     ldflags:
       - -s -w -X github.com/budimanjojo/talhelper/v3/cmd.version={{.Version}}
 archives:
   - name_template: "{{.ProjectName}}_{{.Os}}_{{.Arch}}"
+    id: targz
```

Comment:
Is there a reason why the windows builds should be archived in zip instead of targz?

Reply:
According to [goreleaser docs](https://goreleaser.com/customization/winget/?h=winget) winget archives must be zips, probabbly related to Windows only recently starting to support other archive formats

>     # - 'archive': archives (only if format is zip),